haproxy-ingress/templates/controller-template.yaml

```diff
@@ -7,7 +7,10 @@ metadata:
   name: {{ include "haproxy-ingress.fullname" . }}-template
   namespace: {{ .Release.Namespace }}
 data:
-  haproxy.tmpl: |-
+  haproxy.tmpl: |
+{{- if .Values.controller.templateFile -}}
+{{ .Files.Get .Values.controller.templateFile | indent 4 }}
+{{- else }}
 {{ .Values.controller.template | indent 4 }}
-{{/* .Files.Get "haproxy.tmpl" | indent 4 */}}
+{{- end }}
```

Comment:
these trimleft configs broke the yaml parser, here is a config suggestion that worked for me:

```
  haproxy.tmpl: |
{{- if .Values.controller.templateFile }}
    {{- .Files.Get .Values.controller.templateFile | nindent 4 }}
{{- else }}
    {{- .Values.controller.template | nindent 4 }}
{{- end }}
```

Reply:
fixed